tock/hours/views.py

```diff
@@ -11,10 +11,10 @@
 from django.core.exceptions import ObjectDoesNotExist, ValidationError
 from django.core.urlresolvers import reverse
 from django.http import Http404, HttpResponse
-from django.shortcuts import render
 from django.views.generic import ListView, DetailView, TemplateView
 from django.views.generic.edit import CreateView, UpdateView, FormView
 from django.db.models import Sum
+from django.shortcuts import render_to_response
```

Comment:
Why the reversion to the older render_to_response rather than the newer render() shortcut?